src/Faker/Provider/lb_LU/Address.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Faker\Provider\lb_LU;
+
+class Address extends \Faker\Provider\Address
+{
+    protected static $buildingNumber = array('1##', '##', '#');
+
+    protected static $streetSuffixLong = array(
+        'Gaass', 'Plaz', 'Strooss', 'Str.', 'Wee'
+    );
+
+    protected static $streetSuffixShort = array(
+        'gaass', 'plaz', 'strooss', 'str.', 'wee'
+    );
+
+    protected static $postcode = array('####');
+
+    // Source: https://lb.wikipedia.org/wiki/L%C3%ABscht_vun_de_L%C3%ABtzebuerger_Uertschaften,_H%C3%A4ff_a_Lieu-diten
+    protected static $cityNames = array(
+        'Ärenzdall', 'Bäerdref', 'Bartreng', 'Bauschelt', 'Bech', 'Beefort', 'Beetebuerg', 'Béiwen-Atert', 'Béiwen op der Atert', 'Bettenduerf', 'Betzder', 'Biekerech', 'Bierg', 'Biissen', 'Biwer', 'Buerschent', 'Bus', 'Déifferdeng', 'Diddeleng', 'Dikrech', 'Dippech', 'Duelem', 'Ell', 'Ëlwen', 'Esch-Sauer', 'Esch-Uelzecht', 'Ettelbréck', 'Feelen', 'Fëschbech', 'Fiels', 'Fluessweller', 'Fréiseng', 'Garnech', 'Géisdref', 'Gréiwemaacher', 'Groussbus', 'Habscht', 'Hesper', 'Hiefenech', 'Iechternach', 'Ierpeldeng', 'Jonglënster', 'Käerch', 'Käerjeng', 'Keel', 'Kielen', 'Kiischpelt', 'Klierf', 'Kolmer-Bierg', 'Konsdref', 'Konter', 'Koplescht', 'Leideleng', 'Lenneng', 'Lëntgen', 'Lëtzebuerg', 'Lëtzebuerg (Stad)', 'Luerenzweiler', 'Mäertert', 'Mäerzeg', 'Mamer', 'Manternach', 'Mecher', 'Miedernach', 'Miersch', 'Mompech', 'Monnerech', 'Munneref', 'Nidderaanwen', 'Noumer', 'Parc Housen', 'Péiteng', 'Pëtscht', 'Préizerdaul', 'Rammerech', 'Recken', 'Recken op der Mess', 'Réiden', 'Réimech', 'Reisduerf', 'Réiser', 'Rëmeleng', 'Rouspert', 'Sandweiler', 'Schëffleng', 'Schengen', 'Schëtter', 'Schieren', 'Sëll', 'Simmer', 'Stadbriedemes', 'Steesel', 'Stengefort', 'Stroossen', 'Suessem', 'Tandel', 'Tënten', 'Useldeng', 'Veianen', 'Viichten', 'Wäisswampech', 'Wal', 'Waldbëlleg', 'Waldbriedemes', 'Walfer', 'Wanseler', 'Weiler zum Tuerm', 'Wëntger', 'Wolz', 'Wuermeldeng'
```

Comment:
please break line by first letter to make it more readable. Same for further lists.